index.js

```diff
@@ -96,16 +96,48 @@ function processContentWithPostCSS(options, href) {
 	};
 }
 
-module.exports = function plugin(options) {
+/**
+ * returns object with default settings
+ * @param  {Object} options [plugin's options]
+ * @return {Object}
+ */
+function getDefaultOptions(options) {
 	options = options || {};
 	options.root = path.resolve(options.root || './');
 	options.plugins = options.plugins || [];
 	options.from = options.from || '';
+	options.selectors = options.selectors || {};
```

Comment:
I'm personally in favour of your changes, regardless of being a breaking change 😛 , so a major bump would also be an option imho, if we get 'cleaner' code. @canvaskisa what's your opinion on that?